rosidl_typesupport_introspection_c/include/rosidl_typesupport_introspection_c/message_introspection.h

```diff
@@ -43,6 +43,7 @@ typedef struct rosidl_typesupport_introspection_c__MessageMember
 typedef struct rosidl_typesupport_introspection_c__MessageMembers
 {
   const char * package_name_;
+  const char * message_namespace_;
```

Comment:
Shouldn't the `package_name_` field be removed?

Reply:
Yes, I thought I did that. Oops.
Done, also updated service structs: f5a5db0